freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_backup

```diff
@@ -113,6 +113,30 @@ remove_local_backups() {
     find ${config[backup_path]}/ -name ipa-* -type d  -print0 | xargs -0 /usr/bin/rm -vrf >> $LOGFILE 2>&1 || error_exit "Unable to remove local backup copies"
 }
 
+upload_aws_backup() {
+    echo "try to upload with default encryption"
+    ret_code=$(/usr/bin/aws ${REGION_OPTION} s3 cp --recursive --no-progress ${config[backup_path]}/${BACKUPDIR} ${BACKUP_LOCATION}/${BACKUPDIR} >> $LOGFILE 2>&1 || echo $?)
```

Comment:
If no default encryption is set it will upload it without any encryption.

Reply:
done